app/scripts/metamask-controller.js

```diff
@@ -1794,6 +1806,7 @@ export default class MetamaskController extends EventEmitter {
       PermissionLogController: this.permissionLogController.store,
       SubjectMetadataController: this.subjectMetadataController,
       AnnouncementController: this.announcementController,
+      networksOrderingController: this.networksOrderingController,
```

Comment:
The convention for these keys is for it to be capitalized. Note that this state is referenced on line 769 as `NetworksOrderingController`, but since it's lower-cased here, the persistence is not working

Reply:
Great catch! Yup, this fix makes the state persistent. Updated here https://github.com/MetaMask/metamask-extension/pull/21163/commits/e64a869bc03dd3ce1b12183509ac3de7cee38f12